pkgs/shells/xonsh/default.nix

```diff
@@ -31,15 +31,31 @@ python3Packages.buildPythonApplication rec {
     patchShebangs .
   '';
 
-  doCheck = !stdenv.isDarwin;
+  disabledTests = [
+    "test_loading_correctly"
```

Comment:
Why'd this fail?

Reply:
It fails due sandbox. I updated this bit